envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/containers/ProxyContainer.kt

```diff
@@ -0,0 +1,84 @@
+package pl.allegro.tech.servicemesh.envoycontrol.config.containers
+
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import org.testcontainers.images.builder.ImageFromDockerfile
+import pl.allegro.tech.servicemesh.envoycontrol.testcontainers.GenericContainer
+import java.time.Duration
+
+class ProxyContainer(private val proxyUrl: String) : GenericContainer<ProxyContainer>(
+    ImageFromDockerfile()
+        .withFileFromString("/index.js", """
```

Comment:
It looks so complicated. Maybe you could extract this code to file and just add to container?

Reply:
I can move it to a separate file.